openapi3.yaml

```diff
@@ -245,6 +245,12 @@ paths:
   /jobs/{jobId}/tasks:
     parameters:
       - $ref: '#/components/parameters/jobId'
+      - in: query
```

Comment:
Won't it break the API?
This should be a breaking change to "feat!: xxx"

Reply:
Done